requests/models.py

```diff
@@ -102,7 +102,10 @@ def _encode_params(data):
                         result.append(
                             (k.encode('utf-8') if isinstance(k, str) else k,
                              v.encode('utf-8') if isinstance(v, str) else v))
-            return urlencode(result, doseq=True)
+            if is_py2:
+                return urlencode(result, doseq=True)
+            else:
+                return urlencode(result, doseq=True,quote_via=quote)
```

Comment:
There is a missing space before the comma.

Reply:
Thanks for comments corrected